resources/type-system/inference.md

```diff
@@ -520,23 +544,22 @@ Tt` for some type `Tb` which satisfies schema `Pb`, and some type `Tt` which
 satisfies schema `Pt`.  Constraints in which `X` appears free in either `Pb` or
 `Pt` are ill-formed.
 
+
 ### Closure of type constraints
 
 The closure of a type constraint `Pb <: X <: Pt` with respect to a set of type
 variables `L` is the subtype constraint `Qb <: X :< Qt` where `Qb` is the
 greatest closure of `Pb` with respect to `L`, and `Qt` is the least closure of
 `Pt` with respect to `L`.
 
-XXX this is wrong, fix
-
 Note that the closure of a type constraint implies the original constraint: that
 is, any solution to the original constraint that is closed with respect to `L`,
 is a solution to the new constraint.
 
 The motivation for these operations is that constraint generation may produce a
 constraint on a type variable from an outer scope (say `S`) that refers to a
-type variable from an inner scope (say `T`).  For example, ` <T>(S) -> int <:
-<T>(List<T> -> int ` constrains `List<T>` to be a subtype of `S`.  But this
+type variable from an inner scope (say `T`).  For example, ` <T>(T) -> List<T> <:
+<T>(T -> S) ` constrains `List<T>` to be a subtype of `S`.  But this
```

Comment:
Should it be `<T>(T) -> List<T> <: <T>(T) -> S`?

Reply:
Yes, fixed.